src/Tribe/Views/V2/Hooks.php

```diff
@@ -299,13 +301,25 @@ public function filter_map_canonical_query_args( $map, $url, $rewrite ) {
 	 *
 	 * @return array $classes
 	 */
-	public function filter_body_class( $classes ) {
-		$classes = $this->container->make( Theme_Compatibility::class )->filter_add_body_classes( $classes );
+	public function filter_body_classes( $classes ) {
 		$classes = $this->container->make( Template_Bootstrap::class )->filter_add_body_classes( $classes );
 
 		return $classes;
 	}
 
+	public function add_body_classes() {
+		$classes = $this->container->make( Theme_Compatibility::class )->add_body_classes();
+		$classes = $this->container->make( Template_Bootstrap::class )->add_body_classes();
+	}
+
+	public function body_classes_should_add( $add, $queue ) {
```

Comment:
:no_entry_sign: Missing doc comment for function body_classes_should_add()

<sub>hash: 1128aa8364e8a571b3693c58849eb252</sub>